versioned/storage/rocksdb-proto/src/main/proto/org/projectnessie/versioned/storage/rocksdb/proto/rocksdb.proto

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+syntax = "proto3";
+package nessie.storage.rocksdb;
+
+option java_package = "org.projectnessie.versioned.storage.rocksdb.proto";
+option java_outer_classname = "RocksDBStorageTypes";
+option java_generate_equals_and_hash = true;
+
+message RocksDBObjProto {
```

Comment:
I think we don't need a separate project for this single type.
Just rename it to sth like `GenericObjProto` or so and put it into the existing `-proto` project.